# Dysfunctional seed dispersal in the endemic flora of Madagascar

* Language: R
* Methods: linear regression, multivariate imputation by chained equations (MICE), spatial analyses
* Associated publication: (in prep)

## Are species displaying disfunctional dispersal already threatened with extinction?

In [12]:
pt<-read.csv("PT-data-miceP.csv", header=T,sep=",")
pt <- pt[pt$.imp==0,]
row.names(pt) = pt$.id
pt2 <- subset(pt, select=-.id)
pt2 <- subset(pt2, select=c(Rarity3, IUCN))        # 0=not rare (more than 5 locations), 1=2-5 locations, 2=1 location, 3=0 location (only type specimen)
head(pt2)

Rarity3 IUCN
Abrahamia_buxifolia   0       NA  
Abrahamia_deflexa     0       NA  
Abrahamia_ditimena    0       NA  
Abrahamia_grandidieri 0       LC  
Abrahamia_humbertii   0       NA  
Abrahamia_ibityensis  0       NA

In [14]:
resThreat <- merge(res2, pt2, all.x=FALSE, by="row.names")
head(resThreat)

Row.names             Dispersers Co.occurring_lemurs
1 Abrahamia_buxifolia   15.0        3                 
2 Abrahamia_deflexa     16.0        6                 
3 Abrahamia_ditimena    14.0       14                 
4 Abrahamia_grandidieri 12.0        5                 
5 Abrahamia_humbertii   16.4        1                 
6 Abrahamia_ibityensis  14.0        0                 
  Co.occurring_Disperser_lemurs Rarity3 IUCN
1  3                            0       NA  
2  4                            0       NA  
3 10                            0       NA  
4  2                            0       LC  
5  1                            0       NA  
6  0                            0       NA

##### Percentage of threatened plant species among those without any available lemur disperser 

In [25]:
Threat <- c("VU", "EN", "CR")         # conservation status meaning vulnerable (VU), endangered (EN) and critically endangered (CR)

In [39]:
resThreat_0 <- resThreat[resThreat$Co.occurring_Disperser_lemurs==0, ]
resThreat_0 <- resThreat_0[!is.na(resThreat_0$IUCN),]      # remove species that don't have a conservation status (IUCN = NA)
resThreat_0 <- resThreat_0[!resThreat_0$IUCN=="DD", ]      # remove species considered as Data Deficient by the IUCN (i.e. not enough data available to assess the conservation status)
nrow(resThreat_0)

[1] 100

In [27]:
resThreat_0_T <- resThreat_0[resThreat_0$IUCN%in%Threat,]      # keep only plant species that are threatened
nrow (resThreat_0_T)/nrow(resThreat_0)*100

[1] 93

##### Percentage of threatened plant species in the whole dataset

In [40]:
resThreat2 <- resThreat[!is.na(resThreat$IUCN),]        # remove species that don't have a conservation status (IUCN = NA)
resThreat2 <- resThreat2[!resThreat2$IUCN=="DD", ]      # remove species considered as Data Deficient by the IUCN (i.e. not enough data available to assess the conservation status)
nrow(resThreat2)

[1] 386

In [41]:
resThreat2_T <- resThreat2[resThreat2$IUCN%in%Threat,]      # keep only plant species that are threatened
nrow (resThreat2_T)/nrow(resThreat2)*100

[1] 73.31606

<div class="alert alert-block alert-success">
Among the animal-dispersed plant species __without any available lemur disperser__ (and for which the conservation status has been assessed), __93% are already threatened with extinction__ (i.e. vulnerable, endangered or critically endangered), compared to 73% in the whole dataset (N=386 species with conservation status).

In [54]:
NoStatus <- c("DD", "NA")
resNoStatus <- resThreat[resThreat$IUCN%in%NoStatus,]      # keep only plant species that have no conservation status yet
nrow(resNoStatus)/nrow(resThreat)*100

[1] 0.4638834

In [59]:
resDD <- nrow(subset(resThreat,IUCN=="DD"))
resNA <- sum(is.na(resThreat$IUCN))
resNoStatus <- resDD+resNA
resNoStatus/nrow(resThreat)*100

[1] 87.21007

<div class="alert alert-block alert-success">
Today, the __conservation status__ of __87%__ of Madagascan endozoochorous plant species is __unknown__ which strongly limits the establishment of conservation plans.

## Are species displaying disfunctional dispersal already rare?

##### Percentage of rare plant species among those without any available lemur disperser

In [48]:
Rare <- c(2,3)

In [49]:
resRare_0 <- resThreat[resThreat$Co.occurring_Disperser_lemurs==0, ]
nrow(resRare_0)

[1] 624

In [50]:
resRare_0_R <- resRare_0[resRare_0$Rarity3%in%Rare,]      # keep only plant species that are rare
nrow(resRare_0_R)/nrow(resRare_0)*100

[1] 22.27564

##### Percentage of rare plant species in the whole dataset

In [51]:
resRare2 <- resThreat[!is.na(resThreat$Rarity3),]        # Remove species whose rarity status is unknown
nrow(resRare2)

[1] 3018

In [52]:
resRare2_T <- resRare2[resRare2$Rarity3%in%Rare,]      # keep only plant species that are rare
nrow(resRare2_T)/nrow(resRare2)*100

[1] 9.44334

<div class="alert alert-block alert-success">
__22%__ of the plant species __without any available disperser__ are either __already rare__, with only one recorded location, or even known to be __extinct in the wild__, compared to 9% in the whole dataset (N=3018 species). 